MIP101/MIP101.md

```diff
@@ -1444,31 +1444,31 @@ The Genesis emissions of Allocators are 4.6 billion tokens over 10 years, with t
 
 ###### 4.3.12.2.1: Genesis Farming Emissions
 
-4 billion Allocator tokens are for Genesis farming.
+2.85 billion Allocator tokens are for Genesis farming.
 
 ###### 4.3.12.2.1.1
 
-For the first 2 years, the rate of Genesis farming is 1 billion AllocatorDAO tokens per year
+For the first 2 years, the rate of Genesis farming is 712.5 million AllocatorDAO tokens per year
 
 ###### 4.3.12.2.1.2
 
-For the following 2 years, the rate of Genesis farming is 500 million AllocatorDAO tokens per year.
+For the following 2 years, the rate of Genesis farming is 356.25 million AllocatorDAO tokens per year.
 
 ###### 4.3.12.2.1.3
 
-For the following 2years, the rate of Genesis farming is 250 million AllocatorDAO tokens per year.
+For the following 2 years, the rate of Genesis farming is 178.125 million AllocatorDAO tokens per year.
 
 ###### 4.3.12.2.1.4
 
-For the final 4 years, the rate of Genesis farming is 125 million AllocatorDAO tokens per year.
+For the final 4 years, the rate of Genesis farming is 89.0625 million AllocatorDAO tokens per year.
 
 ###### 4.3.12.2.2
 
-The workforce bonus pool starts with 600 million AllocatorDAO tokens. The Workforce bonus pool can be further topped up through the permanent emissions.
+The workforce bonus pool starts with 600 million AllocatorDAO tokens. The workforce bonus pool can be further topped up through the permanent emissions.
 
-##### 4.3.12.3: AllocatorDAO Permanent Emissions
+###### 4.3.12.2.3
 
-The permanent emissions of AllocatorDAOs are a total of 10% tokens emitted per year.
+An allocation of 1.15b AllocatorDAO tokens will be reserved for MKR holders at the time of subDAO token launch. These tokens have a 10-year linear vesting schedule.
```

Comment:
Let's expand "b" to "billion" here so that it remains consistent with the rest of the text

```suggestion
An allocation of 1.15 billion AllocatorDAO tokens will be reserved for MKR holders at the time of subDAO token launch. These tokens have a 10-year linear vesting schedule.
```